main/handlers.js

```diff
@@ -27,8 +27,14 @@ function installDesktopFile () {
   var desktopFile = fs.readFileSync(templatePath, 'utf8')
 
   desktopFile = desktopFile.replace(/\$APP_NAME/g, config.APP_NAME)
-  desktopFile = desktopFile.replace(/\$APP_PATH/g, path.dirname(process.execPath))
-  desktopFile = desktopFile.replace(/\$EXEC_PATH/g, process.execPath)
+
+  var appPath = config.IS_PRODUCTION ? path.dirname(process.execPath) : config.ROOT_PATH
+  desktopFile = desktopFile.replace(/\$APP_PATH/g, appPath)
+
+  var execPath = process.execPath + (config.IS_PRODUCTION ? '' : ' \.')
```

Comment:
I see, you need to pass the app path into the electron binary for development. Fair enough.

But I don't think this needs to be escaped. It's just a plain string, so `.` should be sufficient. I will test and if this is the case, I will change it.